src/plugins/flow.js

```diff
@@ -726,6 +730,21 @@ pp.typeCastToParameter = function (node) {
   );
 };
 
+pp.flowParseVariance = function() {
+  let variance = null;
+  if (this.match(tt.plusMin)) {
+    variance = this.startNode();
+    if (this.state.value === "+") {
+      variance.kind = "plus";
+    } else if (this.state.value === "-") {
+      variance.kind = "minus";
+    }
+    this.eat(tt.plusMin);
```

Comment:
We could simplify that here and do `this.eat(tt.plusMin)` already in the `if ()` and remove this line then.